spec/lib/metasploit/framework/credential_collection_spec.rb

```diff
@@ -68,7 +68,7 @@
       let(:pass_file) do
```

Comment:
@cgranleese-r7 Not sure why the checks are suddenly failing, I simply changed the structure of the testing file, as we discussed 🤔